src/components/description/Description.js

```diff
@@ -0,0 +1,15 @@
+import HtmlElement from '../htmlElement/HtmlElement';
+import './styles.css';
+
+export class Description extends HtmlElement {
+  constructor(label = '', classList = [], htmlTag = 'p') {
+    super(classList, htmlTag, label);
+  }
+
+  show() {
```

Comment:
I guess `show()` is no longer needed. It seems `Description` and `Header` could be initialised with `render()`.